Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER "Greg Way" <gregway@mail.med.upenn.edu>
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+            libcurl4-openssl-dev \
+            libxml2-dev \
+            ed \
+            git \
+            vim \
+            curl \
+            r-base \
+            r-base-dev \
+            r-recommended
+
+RUN git clone https://github.com/greenelab/gbm_immune_validation.git
```

Comment:
@brettbj does it make sense to include the repo like this in the Docker -- this means that every time you update the codebase you need to build a new Docker image. Is this what you've been doing?